docker_templates/alpine-linux.Dockerfile.j2

```diff
@@ -10,8 +10,10 @@ RUN set -eux; \
         # java.lang.NoClassDefFoundError: Could not initialize class sun.awt.X11FontManager
         # https://github.com/docker-library/openjdk/pull/235#issuecomment-424466077
         fontconfig ttf-dejavu \
+        {% if version|int < 25 -%}
         # gnupg required to verify the signature
         gnupg \
```

Comment:
Why only Java 24 and below?

Reply:
Oh, is this a case that 25+ needs different packages?  worth adding a comment in that case

Reply:
correct, it's JDK25+